### PR TITLE
Add missing return statement in cms_env_set_version()

### PR DESCRIPTION
**Pull request by x64architecture**
This pull request adds a missing return statement in cms_env_set_version().

### CODE REVIEW

**Comment by richsalz** [comment]
+1

File: crypto/cms/cms_env.c [hunk 1]
```diff
@@ -835,8 +835,10 @@ static void cms_env_set_version(CMS_EnvelopedData *env)
     }
     if (env->version == 2)
         return;
-    if (env->originatorInfo || env->unprotectedAttrs)
+    if (env->originatorInfo || env->unprotectedAttrs) {
         env->version = 2;
+        return;
+    }
```
Comment (levitte) [inline crypto/cms/cms_env.c]: Hmmm...  I'd rather see that the original two lines were moved above `if (env->version == 2)`...

**Comment by mattcaswell** [comment]
@x64architecture - there is an outstanding comment for you on this one.

**Comment by richsalz** [comment]
Replaced by #1206

**Pull request closed without merging**